Justfile

```diff
@@ -68,15 +68,15 @@ build-release-artifacts arch nightly="false":
     cargo binstall --no-confirm cross
     cross build --release --target $arch --bin nat-detection $nightly_feature
     cross build --release --target $arch --bin node-launchpad $nightly_feature
-    cross build --release --features network-contacts,websockets --target $arch --bin autonomi $nightly_feature
+    cross build --release --features network-contacts,websockets --target $arch --bin ant $nightly_feature
```

Comment:
Could we merge these changes in a separate PR?

Reply:
Sorry I think that was a previous git commit that somehow snuck in here, it is updated now.